client/src/store/new-request-module.ts

```diff
@@ -1984,8 +2013,9 @@ export class NewRequestModule extends VuexModule {
       }
       this.mutateNameRequest(resp.data)
       this.mutateDisplayedComponent('ExistingRequestDisplay')
-    } catch (error) {
-      console.error('findNameRequest(), error =', error) // eslint-disable-line no-console
+    } catch (err) {
+      const msg = await handleApiError(err, 'Could not find name request')
+      console.error('findNameRequest()', msg) // eslint-disable-line no-console
       this.mutateNameRequest(
```

Comment:
FYI, this is one way errors are handled: overload the NR object with this so the calling page can display an error inline (ie, red alert on the page).